recipes/PySimpleGUIWx/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pysimpleguiwx" %}
+{% set version = "0.15.0" %}
+{% set styledName = "PySimpleGUIWx" %}
```

Comment:
FYI, you don't need to use `styledName` - you can have `name = "PySimpleGUIWx` and use `name|lower` when needed.